test/unix/force_merge.ml

```diff
@@ -25,49 +25,64 @@ let test_one_entry r k v =
       Alcotest.failf "Inserted value is not present anymore: %s." k
 
 let test_fd () =
+  let ( >>= ) x f = match x with `Ok x -> f x | err -> err in
```

Comment:
Can you pick another infix operator, just to ease the potential migration to `lwt` (if that ever happens).

Reply:
This function doesn't actually use the Index API at all, but have switched to `>>?` regardless.